src/security_context/BCB_AES_GCM.c

```diff
@@ -384,7 +402,7 @@ int BSLX_BCB_GetParams(const BSL_BundleRef_t *bundle, BSLX_BCB_t *bcb_context, c
         {
             case RFC9173_BCB_SECPARAM_IV:
             {
-                assert(!is_int);
```

Comment:
Thank you for removing bare `assert()`. If you notice any others please write them up at least.